examples/icon-switching/main_linux.go

```diff
@@ -0,0 +1,53 @@
+// This example shows you how to change the icon in the webview on Linux
+
+package main
+
+/*
+#cgo linux pkg-config: gtk+-3.0
+
+#include <gtk/gtk.h>
+
+void set_external_icon(void *ptr, char* iconPath) {
+	gtk_window_set_icon_from_file(ptr, iconPath, NULL);
+}
+*/
+import "C"
+import (
+	"time"
+	"unsafe"
+
+	"github.com/webview/webview"
+)
+
+// Set icon from external file
+func setExternalIcon(w webview.WebView, iconPath string) {
+
+	hwnd := w.Window()
+	cstr := C.CString(iconPath)
+
+	defer C.free(unsafe.Pointer(cstr))
+
+	C.set_external_icon(hwnd, cstr)
```

Comment:
Change `hwnd` to `gtk_window` for clarity.

Reply:
Done